modules/wsl-conf.nix

```diff
@@ -41,6 +41,21 @@ with lib; {
         default = false;
         description = "Use systemd as init. There's no need to enable this manually, use the wsl.nativeSystemd option instead";
       };
+      initPath = mkOption {
+        type = path;
```

Comment:
Doesn't this need to be ``nullOr path``?

Reply:
Probably, I haven't actually tested this on non-native-systemd yet.